### PR TITLE
Improvement: Update frontend styles to match Figma design

### PR DESCRIPTION
**Pull request by KiaanCastillo**
# Changes
- Change theme to light 
- Update layout and styles of products page, product page, and cart page
   - Create components for modularization
- Add `.vscode` to `.gitignore` 
- Add "lh3.googleusercontent.com" to list of acceptable image domains (for using image URLs from publicly viewable image from a Google Drive folder)

## Figma Design Mockups
https://www.figma.com/file/9AsJupryfPceLAoarhHqZe/SSSS-Website?type=design&node-id=1793%3A4097&mode=design&t=4MqkNXwZ7MmBPDyG-1

### CODE REVIEW

**Comment by brianrahadi** [comment]
Looking good!! :D, I will init the environment and check how it looks like locally later

**Comment by marcusgchan** [comment]
Looks really good!

I'm not sure if this is intentional or not but the on hover styles for the add to cart button is a bit different from the place order button.

image

image

File: src/components/products/Header.tsx [hunk 1]
```diff
@@ -1,24 +1,30 @@
...
-            </li>
-          </ul>
-        </nav>
-      </div>
+    <header className="flex justify-between px-4 py-7">
+      <Link href="/products">
+        <Logo />
+      </Link>
+      <nav>
+        <ul className="flex gap-9">
```
Comment (brianrahadi) [inline src/components/products/Header.tsx:12]: nit: Should we ditch `ul` and just use `div` instead since children is `Link`?

**Review by brianrahadi — APPROVED** [review]

File: src/components/products/Header.tsx (hunk 1 above)
Comment (marcusgchan) [reply src/components/products/Header.tsx:12]: Yea, I think either could work. I think for screen readers it'll read out the number of items which can be useful if it's a list but it's prob easier to change the ul to a div.

**Review by marcusgchan — APPROVED** [review]

**Review by marcusgchan — APPROVED** [review]
I'm just gonna merge this in now

**Pull request merged**